_posts/blog/2016-05-16-animations_touch.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title: "Animations and Touch Events"
+modified:
+categories: blog
+author: florina_muntenescu
+excerpt: Android offers two main animation frameworks - view animation and property animation. The main difference between them can be seen when you need to handle touch events on views that have changed their position after animation.
+tags: [Android, Animations, Touch Events]
+image:
+date: 2016-05-15T15:39:55-04:00
+---
+
+Android offers two main animation frameworks: **view animation** and **property animation**. Each of these is fairly easy to implement. However, the main difference can be seen when you need to handle touch events on views that have changed their position after animation. Let’s see how both of these animation frameworks work, how they can be implemented and which one you should use for touch events.
+
+##Task
+When tapping a ``View``, we need to translate it up by 75% of its height. When tapping on the area where the ``View`` was initially, our ``View`` should slide back down.
+
+<picture>
+	<img src="/images/animations_touch/animations_touch_slide_up_down.gif" alt="image">
+</picture>
+
+##View Animations
+
+**View animations** provide an easy to use API for animating the contents of a ``View`` object. The animations supported are translations, rotations, growing and shrinking of a view. You can define these animations either in XML or programmatically.
```

Comment:
How about instead of "The animations supported are translations" change it to "Framework supports animations: translation, rotation, growing" and so on